mailpoet/lib/Config/Changelog.php

```diff
@@ -141,7 +161,15 @@ private function checkWooCommerceListImportPage() {
 
   private function checkRevenueTrackingPermissionPage() {
     if (
-      !in_array($_GET['page'], ['mailpoet-woocommerce-setup', 'mailpoet-welcome-wizard', 'mailpoet-migration'])
+      !isset($_GET['page']) ||
+      !in_array(
+        sanitize_text_field(wp_unslash($_GET['page'])),
+        [
+          'mailpoet-woocommerce-setup',
+          'mailpoet-welcome-wizard',
+          'mailpoet-migration',
+        ]
+      )
```

Comment:
Just a stylistic note that we may not need to fix now — we seem to repeat `sanitize_text_field(wp_unslash($_GET['page']))` a lot here so it may be worth extracting that into a private method.

Probably applies to some other files in this commit as well.